### PR TITLE
Grammar and formatting suggestions for Section 5, Basic Concepts

### PR DESCRIPTION
**Pull request by grantnoble**
Again, mostly grammar and formatting suggestions, but please pay special attention to the Object/Type table in section 5.3 Types. I made some significant changes to the format of this table so I hope it's ok.

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/297.html" title="Last updated on Nov 30, 2018, 8:25 AM GMT (5e8948c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/297/8b58476...grantnoble:5e8948c.html" title="Last updated on Nov 30, 2018, 8:25 AM GMT (5e8948c)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -950,66 +953,68 @@ <h2>Types</h2>
...
+Issuer information can be expressed using the following <a>properties</a>:
         </p>
         <dl>
           <dt><var>issuer</var></dt>
           <dd>
-The value of this property MUST be a URI. It is RECOMMENDED that dereferencing
-the URI results in a document containing machine-readable information about
-the issuer that may be used to verify the information expressed in the
-<a>credential</a>.
+The value of the <code>issuer</a> property MUST be a URI. It is RECOMMENDED
```
Comment (davidlehn) [inline index.html]: Fix mismatched tag.

Comment (grantnoble) [reply index.html]: Done.

File: index.html [hunk 2]
```diff
@@ -884,64 +883,68 @@ <h2>Types</h2>
...
           <tbody>
             <tr>
               <td>
-A <a href="#credentials">credential</a> object
+<a href="#credentials">Credential</a>&nbsp;object
               </td>
               <td>
-<tt>"type": ["<code>VerifiableCredential</code>"]</tt> and
-a more specific credential type (e.g., <code>ProofOfAgeCredential</code>)
+<code>VerifiableCredential</code> followed by a more specific
```
[the remark this answers is not in the document]
Comment (grantnoble) [reply index.html]: Done for both Verifiable Credential and Verifiable Presentation.

File: index.html [hunk 3]
```diff
@@ -1032,20 +1037,20 @@ <h2>Issuer</h2>
       </section>
 
       <section>
-        <h2>Proofs (e.g., Signatures)</h2>
+        <h2>Proofs (Signatures)</h2>
```
Comment (davidlehn) [inline index.html]: The Verification "Proofs" section has the "e.g.". They should probably do the same thing one way or the other.

File: index.html [hunk 4]
```diff
@@ -823,46 +825,43 @@ <h2>Identifiers</h2>
...
-<a>credential</a> or <a>presentation</a> is appropriate.
-Type information MUST be expressed via the <a>type</a> property:
+Software systems that process the kinds of objects specified in this document
+use type information to determine whether or not a provided <a>credential</a> 
+or <a>presentation</a> is appropriate. Type information MUST be expressed 
+using the <code>type</code> property.
         </p>
 
         <dl>
           <dt><dfn>type</dfn></dt>
```
Comment (davidlehn) [inline index.html]:
```suggestion
           <dt><dfn data-lt="type|types">type</dfn></dt>
```
How about adding the plural here so `<a>types</a>` can be used vs `<a>type</a>s`.  If this is added could fix the various places it's used.

File: index.html [hunk 5]
```diff
@@ -950,66 +953,68 @@ <h2>Types</h2>
...
+process this additional information.
+        </p>
+
+        <p>
+When processing encapsulated objects defined in this specification, (for 
+example, objects associated with the <code>credentialSubject</code> object 
+or deeply nested therein), software systems SHOULD use the <a>type</a> 
+information specified in encapsulating objects higher in the hierarchy. 
+Specifically, an encapsulating object, such as <a>credential</a>, SHOULD 
+convey the associated object <a>types</a> so that <a>verifiers</a> can quickly
```
Comment (davidlehn) [inline index.html]: This `types` link and the one below are not showing up (and the commit diff previewer seems confused by them).  See the `dfn` comment for a fix.

Comment (grantnoble) [reply index.html]: Great suggestion!

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Yes, @davidlehn. My intended upcoming review of Section 6 Advanced Topics will suggest the same change.

File: index.html [hunk 6]
```diff
@@ -950,66 +953,68 @@ <h2>Types</h2>
...
-To provide a concrete example, a <a>credential</a> with the additional
-<code>type</code> of <code>ProofOfAgeCredential</code> would signal to the
-<a>verifier</a> that the object associated with the <a>credentialSubject</a> property will
-contain the identifier for the <a>subject</a> in the <code>id</code> property
-and the age assertion in the <code>ageOver</code> property. This enables
-implementers to rely on values associated with the <code>type</code> property
-for verification purposes. The expectation of types and their associated
-properties SHOULD be documented in at least a human-readable specification
-and, preferably, in an additional machine-readable representation.
+specify, or be associated with, additional more narrow <a>type</a>s (like
```
Comment (davidlehn) [inline index.html]: With the dfn fix this can now be less awkward:
```suggestion
specify, or be associated with, additional more narrow <a>types</a> (like 
```

Comment (grantnoble) [reply index.html]: Done just now! :)

File: index.html [hunk 7]
```diff
@@ -1180,25 +1185,25 @@ <h2>Status</h2>
...
-        <h2>Presentations</h2>
+        <h3>Presentations</h3>
 
         <p>
-Credentials MAY be used to construct a <a>verifiable presentation</a>. A
-verifiable presentation may be of a verifiable credential itself, or
-of data derived from verifiable credentials in a format that is
-cryptographically verifiable.
+<a>Credentials</a> MAY be used to construct a <a>verifiable presentation</a>. 
+A <a>verifiable presentation</a> can be of a <a>verifiable credential</a>
```
Comment (David-Chadwick) [inline index.html]: We have the same two conformance issues here. 
1. Presentations MAY be used to combine VCs.
2. If Presentations are used, then they MUST contain VCs. 
i.e. the use of presentations is optional, but the format of presentations is standardised.

File: index.html [hunk 8]
```diff
@@ -1134,26 +1139,26 @@ <h2>Expiration</h2>
...
+<code>credentialStatus</code> <a>property</a>. This property comprises the 
+type of <a>credential</a> status information being provided (sometimes 
+referred to as the credential status scheme), and the <a>id</a> of the status
 type instance. The precise contents of the credential status information is
-           determined by the specific <code>credentialStatus</code> type definition,
-    and will vary
- depending upon a variety of factors, such as whether it is
-            simple to implement or privacy-enhancing.
+determined by the specific <code>credentialStatus</code> type definition,
+and varies depending on factors like whether it is simple to implement or
```
Comment (David-Chadwick) [inline index.html]: factors like whether -> factors such as whether

File: index.html [hunk 9]
```diff
@@ -1134,26 +1139,26 @@ <h2>Expiration</h2>
...
-        such as suspension or revocation,
-        may be provided by adding the <code>credentialStatus</code> <a>property</a>.
-This property
-       comprises the type of credential status information that is being provided
- (sometimes referred to as the credential status scheme), plus the id of the status
+        <h3>Status</h3>
+
+        <p>
+Information about the current status of a <a>verifiable credential</a>, such 
+as whether it is suspended or revoked, can be provided by adding the
```
Comment (David-Chadwick) [inline index.html]: Note that there are two conformance issues here
1. The Status is optional, so it MAY be provided in a VC
2. The contents of the Status, if present, MUST conform to that specified here
So we need to reword this to ensure that both conformance statements are present. 

Information about the current status of a VC,...., MAY be provided by adding.....

We already have the MUST statement later on when credentialStatus is defined

N.B. This is a recurring problem in this document, as it tries to combine the two conformance statements into one, which leads to confusion.

Comment (grantnoble) [reply index.html]: Done. I split the two requirements using bullet points, which seemed an easier way to present it than in para form.

Comment (David-Chadwick) [reply index.html]: Yes fine thanks
